cmake/Corrosion.cmake

```diff
@@ -388,79 +395,81 @@ function(_corrosion_add_library_target)
     set("${CALT_OUT_ARCHIVE_OUTPUT_BYPRODUCTS}" "${archive_output_byproducts}" PARENT_SCOPE)
 
     if(has_staticlib)
-        add_library(${target_name}-static STATIC IMPORTED GLOBAL)
-        add_dependencies(${target_name}-static cargo-build_${target_name})
-        set_target_properties(${target_name}-static PROPERTIES COR_FILE_NAME ${static_lib_name})
+        add_library(${target_name_cmake_underscore}-static STATIC IMPORTED GLOBAL)
```

Comment:
Is there a reason we don't use `target_name_cmake` / `::` for the internal library name?